### ISSUE TITLE
Mixins execute backwards

### ISSUE THREAD

**Issue by jonathanazulay** [meteor/validated-method#57]
In my application I do something like this

`mixins: [LoggedInMixin, UserIsBlaBla]`

The second mixin in the array depends on something in Meteor.user() so we need to make sure the user is logged in first. But, unexpectedly, mixins will run in reverse order so I have to reverse the array:

`mixins: [UserIsBlaBla, LoggedInMixin]`

**Comment by stubailo** [comment]
You're right! We do a `forEach` over the mixins, and so if they wrap the run function it goes like:

```
// First:
loggedIn(run())

// Second:
userIsBlaBla(loggedIn(run))
```

But I think we pretty much can't change it now since that would probably break many things, so would you mind submitting a PR to the docs that reminds people of this?

**Comment by jonathanazulay** [comment]
Oh I see. If you think about how it actually works - that mixins just wrap run() it makes sense. You could argue that just forEach-iterating is the right way aswell since mixins likewise could be dependent on properties added to the methodOptions in a earlier mixin. So never mind this then.

I'll submit a docs PR this week 👍

**Comment by abernix** [comment]
@jonathanazulay I realize this is quite old but, did this doc update PR ever occur somewhere?
